features/GRD000_Grid-information.feature

```diff
@@ -0,0 +1,26 @@
+@implementer-agreement
+@GRD
+@POS
+@version1
+@E00020
+
+Feature: GRD000 - Grid Information
+    The rule verifies the presence of IFC entities used to define a design grid to be used as reference for object placement.
+    https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/concepts/Product_Shape/Product_Placement/Product_Grid_Placement/content.html
+
+
+    Scenario: Check for activation
+
+    Given an IfcGridPlacement
+    Given its attribute PlacesObject
+    Given its entity type is 'IfcProduct' including subtypes
+    Given return to IfcGridPlacement
+    Given its attribute PlacementRelTo
+    Given its attribute PlacesObject
+    Given its entity type is 'IfcGrid' 
```

Comment:
These three lines should actually not be a given statement, but a then statement.

    Given an IfcGridPlacement Then PlacementRelTo/PlacesObject **must be** IfcGrid

Reply:
This makes it a normative rule, and not a rule to check for activation. 
Which is part of the reason why this PR is de-activated. It would probably be better to create two new normative rules and then skip the 000-rule. Based on [this issue](https://github.com/buildingSMART/ifc-gherkin-rules/issues/124)